include/unifex/with_scheduler_affinity.hpp

```diff
@@ -144,7 +157,7 @@ struct _fn final {
     if constexpr (
         !same_as<blocking_kind, blocking_t> &&
         (blocking_kind::always_inline == blocking_t{})) {
-      return Awaitable{(Awaitable &&) awaitable};
+      return Awaitable{(Awaitable&&)awaitable};
```

Comment:
nit: wanna rid of the C-style cast?

Reply:
I do want to, but the clock has run out.